keystorage/src/main/java/tech/pegasys/web3signer/keystorage/azure/AzureKeyVault.java

```diff
@@ -23,42 +26,48 @@
 import java.util.function.BiFunction;
 
 import com.azure.core.credential.TokenCredential;
+import com.azure.core.credential.TokenRequestContext;
 import com.azure.core.exception.ResourceNotFoundException;
 import com.azure.core.http.rest.PagedIterable;
 import com.azure.identity.ClientSecretCredentialBuilder;
 import com.azure.identity.ManagedIdentityCredentialBuilder;
 import com.azure.security.keyvault.keys.KeyClient;
 import com.azure.security.keyvault.keys.KeyClientBuilder;
+import com.azure.security.keyvault.keys.KeyServiceVersion;
 import com.azure.security.keyvault.keys.cryptography.CryptographyClient;
 import com.azure.security.keyvault.keys.cryptography.CryptographyClientBuilder;
+import com.azure.security.keyvault.keys.cryptography.models.SignatureAlgorithm;
 import com.azure.security.keyvault.keys.models.KeyVaultKey;
 import com.azure.security.keyvault.secrets.SecretClient;
 import com.azure.security.keyvault.secrets.SecretClientBuilder;
 import com.azure.security.keyvault.secrets.models.KeyVaultSecret;
 import com.azure.security.keyvault.secrets.models.SecretProperties;
+import io.vertx.core.json.JsonObject;
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
+import org.apache.tuweni.bytes.Bytes;
 
 public class AzureKeyVault {
 
   private static final Logger LOG = LogManager.getLogger();
-
   private final TokenCredential tokenCredential;
   private final SecretClient secretClient;
   private final KeyClient keyClient;
+  private static final List<String> SCOPE = List.of("https://vault.azure.net/.default");
```

Comment:
This looks odd to me, is it an azure vault convention? Do we need to make it configurable?

Reply:
This is the recommended by Azure. Obviously this depends on what service you are trying to access. For instance in their example they use `https://graph.microsoft.com/.default` for the graph service. In our case think it's safe to hardcode that to vault since it's the only service our application will use the REST API for.